master/internal/config/resource_manager_config.go

```diff
@@ -106,7 +104,6 @@ type KubernetesResourceManagerConfig struct {
 	DefaultScheduler         string                  `json:"default_scheduler"`
 	SlotType                 device.Type             `json:"slot_type"`
 	SlotResourceRequests     PodSlotResourceRequests `json:"slot_resource_requests"`
-	Fluent                   FluentConfig            `json:"fluent"`
```

Comment:
just double checking, you made sure the master is fine with extra configs, right?

Reply:
After battling GKE, gotta love when the Chart.yaml points at something that doesn't exist causing master to crash because it can't pull the determined repo), master fails when this isn't present. 

I went ahead and reverted the changes on the `resource_manager_config.go` and tested in a GKE cluster

Reply:
Ah, ok. I tested it locally, see the same. Dang it.